plugins/named-containers/post-deploy

```diff
@@ -29,9 +30,12 @@ named_containers_post_deploy() {
     fi
     local ID=$(cat "$container")
     local CURRENT_NAME=$(docker inspect -f '{{.Name}}' "$ID" | tr -d /)
-    dokku_log_info2_quiet "renaming container (${ID:0:12}) $CURRENT_NAME to $NAME"
-    docker rename "$CURRENT_NAME" "$NAME" > /dev/null
+    if [[ $CURRENT_NAME ]]; then
```

Comment:
That is the reverse logic. Assuming you meant -n "$CURRENT_NAME", that and what I did are equivalent. I'm happy to use -n if you feel strongly. I think we've not settle on a standard there.